cognite_toolkit/_cdf_tk/commands/auth.py

```diff
@@ -266,25 +267,27 @@ def _create_toolkit_group_in_cdf_interactive(
                 f"Would have created group {toolkit_group.name!r} with {len(toolkit_group.capabilities or [])} capabilities."
             )
             return None
-
-        while True:
-            source_id = questionary.text(
-                "What is the source id for the new group (typically a group id in the identity provider)?"
-            ).ask()
-            if source_id:
-                break
-            print("Source id cannot be empty.")
-
-        toolkit_group.source_id = source_id
-        if already_used := [group.name for group in all_groups if group.source_id == source_id]:
-            self.warn(
-                HighSeverityWarning(
-                    f"The source id {source_id!r} is already used by the groups: {humanize_collection(already_used)!r}."
+        auth_vars = AuthVariables.from_env()
```

Comment:
This assumes that the credentials you are currently using are the ones you should use. I think this flow should be more explicit. Instead, ask first which provider you are using for creating the Toolkit group.